### ISSUE TITLE
[Feature Request]: Meshtasticd connected node not displaying chUtil and AirUilTX

### ISSUE THREAD

**Issue by ddave47** [meshtastic/firmware#3657]
### Platform

NRF52, ESP32

### Description

When connected to a meshtasticd node, the app is not displaying chUtil and AirUilTX for that specific node.

**Comment by madeofstown** [comment]
Yeah, that feature was working about a month ago (I swear!) but seems that recent updates regressed things. 
Would also be nice if Meshtasticd could show something other than the battery with a question mark on it. Maybe needs to default to USB powered (🔌)?

**Comment by fifieldt** [comment]
@vidplace7  - have you seen chUtil and AirUilTX  on meshtasticd recently?

**Comment by fifieldt** [comment]
I setup meshtasticd and can confirm this is now working.
